wext/mcmc.py

```diff
@@ -45,19 +47,21 @@ def _collection_weight(collection):
         return sum( _weight(M) for M in collection )
 
     def _to_collection(solution):
-        return frozenset( frozenset(M) for M in solution.values() )
+        return frozenset( frozenset(M) for M in solution.values() ) 
 
     # Compute the acceptance ratio
-    def _log_accept_ratio( W_current, W_next ): return W_next - W_current
+    def _log_accept_ratio( W_current, W_next ): 
+        return W_next - W_current
 
     # Set up PRNG, sample space, and output
     random_seed(seed)
     t          = len(ks)
-    genespace  = geneToCases.keys()
+    genespace  = list(geneToCases.keys())
     setsToFreq = [ defaultdict(int) for _ in xrange(nchains) ]
     setToPval, setToObs =  dict(), dict()
     for c in xrange(nchains):
```

Comment:
I'm trying to avoid `ranges()` for code which is both Python2.x and 3.x compatible (cf discussion of performance/memory above), so I went with a `list()` here. 

Is it clear why `range()` is a better choice than `list()` above?